src/main/scala/devices/debug/Periphery.scala

```diff
@@ -330,7 +324,7 @@ object Debug {
       }
 
       debug.apb.foreach { apb =>
-        apb.tieoff()
+        //apb.tieoff()
```

Comment:
I think just tieoff by hand here (Not a fan of `.tieoff` methods that infer direction)